### PR TITLE
remove useless boolean expressions

### PR DESCRIPTION
**Pull request by KisaragiEffective**
This is a kind of refactoring; modification was done automatically by Intellij IDEA.

### CODE REVIEW

File: DynmapCore/src/main/java/org/dynmap/utils/ForgeConfigFile.java [hunk 1]
```diff
@@ -80,7 +80,7 @@ else if (Character.isWhitespace(c)) {
                                 instr = intok = false;
                                 break;
                             case '{':
```
Comment (KisaragiEffective) [inline DynmapCore/src/main/java/org/dynmap/utils/ForgeConfigFile.java:82]: note

**Comment by mikeprimm** [comment]
I don't accept style PRs

**Comment by KisaragiEffective** [comment]
Have fun with your **OWN** technical debt :zany_face:

**Comment by mikeprimm** [comment]
LOL - impllies functional consquence, and there is none here (it's not JavaScript...).  Good luck with your pedantic and valueless PRs!

**Pull request closed without merging**